docs/User/Project/Upgrading.md

```diff
@@ -94,7 +94,7 @@ The `CrossStream` type in `Streamly.Internal.Data.Stream` is a
 wrapper over `Stream` type supplying the Monad instance. However, see
 the "Stream Fusion" section in the `Streamly.Data.Stream` module for
 limitations of the `Stream` type. The `StreamK` type and `CrossStreamK`
-(in Streamly.Internal.Data.Stream.StreamK) could be used to overcome the
+(in Streamly.Internal.Data.StreamK) could be used to overcome the
```

Comment:
This guide is to upgrade from 0.8 to 0.9, therefore, should not reflect changes in 0.10 .